app/clients/sms/aws_sns.py

```diff
@@ -48,7 +49,7 @@ def _valid_sender_number(self, sender):
 
     def send_sms(self, to, content, reference, sender=None, international=False):
         matched = False
-
+        print(hilite(f"TO {to}"))
```

Comment:
`print()` statement. Maybe change to `current_app.logger`?

Reply:
whoops that's just an oversight, good catch.